hack/vendor.sh

```diff
@@ -57,6 +57,8 @@ clone git github.com/Sirupsen/logrus v0.7.2
 
 clone git github.com/go-fsnotify/fsnotify v1.0.4
 
+clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
```

Comment:
I'm +1 for vendoring (we might use this is unit tests in the future, and those can *often* run on the host)